graphcanvas/tests/test_graph_view.py

```diff
@@ -16,18 +18,18 @@
 
 
 class DummyHasTraitsObject(HasTraits):
-    label = Str
+    label = Unicode
 
 
 class TestGraphFromDict(unittest.TestCase):
 
     def test_graph_from_dict(self):
         d = {'a':['b'], 'b':['c', 'd'], 'c':[], 'd':[], 'e':['d']}
         g = graph_from_dict(d)
-        for key, value in d.iteritems():
+        for key, value in d.items():
             children = g.successors(key)
             expected_children = value
-            self.assertListEqual(children, expected_children)
+            self.assertListEqual(sorted(children), sorted(expected_children))
```

Comment:
Can you do a `self.assertItemsEqual` here (or `self.assertCountEqual` on Python 3) to test for equality up to ordering?